css/style.css

```
@@ -198,4 +206,155 @@ footer {
   color: #FFFFFF;
   text-decoration: none;
   margin: 0 10px;
+}
+
+.sns {
+  display: flex;
+  gap: 10px;
+}
+
+
+/* Media Query */
+
+/* Tablet: 768px 이상 ~ 1199px 이하 */
+@media (max-width: 1199px) {
+  .banner {
+    padding: 0 24px;
+  }
+
+  .top-section,
+  .bottom-section {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 480px;
+  }
+
+  .top-content,
+  .bottom-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-end;
+    text-align: center;
+  }
+
+  .top-title h2,
+  .bottom-title h2 {

```

❗️ 수정요청
상단에서 해당 요소들의 height가 고정값 50px로 지정되어 있어 버튼이 해당 글자를 가리네요.
가능하면 상단에서 height: 50px를 제거하시는 것을 추천드려요!
필요한 속성이라면 지금 반응형에서 아래처럼 height: auto를 추가해주세요.

```suggestion
  .top-title h2,
  .bottom-title h2 {
    height: auto;
```